themes/morphosource/css/morphosource.css

```diff
@@ -1221,9 +1221,6 @@ input.lookupBg, textarea.lookupBg, #formArea input.lookupBg{
 }
 
 #mediaDetail #mediaDetailImageScrollArea{
```

Comment:
if this is important for the fix, consider adding a query string to the css include to force the browser to load the updated css.